components/playlist/browser/resources/api/playerEventsNotifier.ts

```diff
@@ -40,8 +44,11 @@ export type PlayerEventsPayload =
   | ({
       type: types.PLAYLIST_OPEN_SOURCE_PAGE
     } & OpenSourcePagePayload)
+  | ({
```

Comment:
Just curious (you don't need to change anything) but is there a reason we keep the `type` and the payload definition separate, rather than doing something like:

```ts
type GoBackPayload = {
    type: types.PLAYLIST_GO_BACK_TO_CURRENTLY_PLAYING_FOLDER,
    data: { ... }
}

type FailedToPlayItemPayload = {
    type: types.PLAYLIST_PLAYER_FAILED_TO_PLAY_ITEM
    data: PlaylistItem
}

type PlayerEventsPayload = GoBackPayload | FailedToPlayItemPayload | ...
```

Reply:
Good point, somehow I thought I should separate type and data 😄